.travis.yml

```
@@ -39,6 +39,8 @@ before_install:
 install:
   # install everything for full dev in the fxa-content-server.
   - travis_retry npm install --silent
+  - npm ls --depth 0 | sed -e ':a' -e 'N' -e '$!ba' -e 's/\n/ /g'
+  - grunt lint
```

Why did we move these earlier? Fail faster?